lib/volley/in_order_subscription.ex

```diff
@@ -388,9 +412,30 @@ defmodule Volley.InOrderSubscription do
 
   @impl GenStage
   def handle_info(:subscribe, state) do
+    handle_demand(1, %__MODULE__{state | producing?: true})
+  end
```

Comment:
I think this is kinda clever

just changing the flag in state won't immediately goose the consumer into getting the events it demands, so we invoke the handle_demand callback which may dump some events into the producer buffer, which will be pushed to the consumer(s)